server/src/main/scala/com/cloudera/livy/utils/LineBufferedStream.scala

```diff
@@ -37,16 +39,16 @@ class LineBufferedStream(inputStream: InputStream) extends Logging {
     override def run() = {
       val lines = Source.fromInputStream(inputStream).getLines()
       for (line <- lines) {
+        info(s"stdout: $line")
         _lock.lock()
-        try {
-          _lines = _lines :+ line
+         try {
+          _lines add line
```

Comment:
Remove the prefix whitespace before `try`. Also be better to use `_lines.add(line)` for clearance.